### ISSUE TITLE
Context Encoder

### ISSUE THREAD

**Issue by fsnelgar** [GONGJIA0208/Diffpose#7]
In the paper a 'Context Encoder' is used to condition the diffusion model. 
It appears that this conditioning is here https://github.com/GONGJIA0208/Diffpose/blob/main/models/gcndiff.py#L101 but doesn't appear to be used. Is this the case?

**Comment by GoogQing** [comment]
Yes, we just use 'u v' as the condition in the frame-based experiment, which is strong enough.
The feature from the context encoder is mainly used for video-based experiments which require dense spatial-temporal information.

When we have time in recent months, we will update the model for the video-based pipeline. Thanks :)

**Comment by gtftyih** [comment]
> 是的，我们只是在基于框架的实验中使用“u v”作为条件，这已经足够强大了。上下文编码器的功能主要用于需要密集时空信息的基于视频的实验。
> 
> 最近几个月有时间时，我们将更新基于视频的管道的模型。谢谢:)



> Yes, we just use 'u v' as the condition in the frame-based experiment, which is strong enough. The feature from the context encoder is mainly used for video-based experiments which require dense spatial-temporal information.
> 
> When we have time in recent months, we will update the model for the video-based pipeline. Thanks :)

Hello，does ‘GCNpose’ just get the 3D pose directly?
